### PR TITLE
error proofing

### PR DESCRIPTION
**Pull request by wkev**

### CODE REVIEW

**Comment by Cat-who-loves-bots** [comment]
not really needed

File: beemo/commands/help/help.js [hunk 1]
```diff
@@ -86,10 +86,10 @@ module.exports = {
                 embed.addField(category, categoryCommands, true);
             }
 
-            message.author.sendEmbed(embed);
+            message.author.sendEmbed(embed).then(() => message.channel.sendMessage("Check your direct messages for a list of commands!")).catch(() => message.channel.sendEmbed(embed));
```
Comment (Cat-who-loves-bots) [inline beemo/commands/help/help.js]: async/await please

Comment (itslukej) [inline beemo/commands/help/help.js]: works fine there cate

Comment (wkev) [inline beemo/commands/help/help.js]: not needed

**Pull request merged**